client-v2/src/services/faciaApi.js

```diff
@@ -65,6 +65,26 @@ async function fetchLastPressed(frontId: string): Promise<string> {
     });
 }
 
+async function publishCollection(collectionId: string): Promise<void> {
+  // The server does not respond with JSON
+  try {
+    await pandaFetch(`/collection/publish/${collectionId}`, {
```

Comment:
Can we not just return this promise directly rather than a void promise created by the async function, just in the event that we start returning something useful for the response? Not a deal breaker, just a thought ....

Reply:
Yeah, this felt a bit weird but so did returning the promise because I don't use it. I guess we can return it when we actually end up doing something with it.